caim_base/models/fosterer.py

```diff
@@ -9,11 +9,11 @@
 from reportlab.pdfbase.pdfmetrics import stringWidth
 from reportlab.pdfgen import canvas
 
-from caim_base.utils import (draw_pdf_checkbox, draw_pdf_choices,
-                             draw_pdf_divider, draw_pdf_field, draw_pdf_header,
-                             draw_pdf_label, draw_pdf_long_field,
-                             draw_pdf_page_num, draw_pdf_title, font_name,
-                             header_font_name, init_pdf)
+# from caim_base.utils import (draw_pdf_checkbox, draw_pdf_choices,
```

Comment:
@rwwiv these failed to import. When I comment them out, the page works. I don't see this file in your branch. So I am commenting them out for the moment. LMK on slack if I can just delete this forever